common/changes/office-ui-fabric-react/keco-detailscolumn-nulls_2019-02-14-01-35.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "DetailsColumn: Remove unnecessary null assertions",
+      "type": "patch"
```

Comment:
can probably be none since the changes are purely in TS and not in emitted JS, but not that important